pkg/windows/dockerfile

```diff
@@ -54,7 +54,7 @@ RUN powershell.exe -Command pip install -r pyinstaller-requirements.txt;
   
 #Move portable git to a new location
 RUN powershell.exe -Command New-Item C:/temp/hubble/PortableGit -ItemType Directory; \
-  Copy-Item -Path C:/tools/git -Destination C:/temp/hubble/PortableGit -Recurse;
+  Copy-Item -Path C:/tools/git/* -Destination C:/temp/hubble/PortableGit -Recurse;
```

Comment:
In my tests you need to have a trailing / after PortableGit or it doesnt work. This should be added here just in case